src/opers.c

```diff
@@ -1823,61 +1824,25 @@ Obj FuncCOMPACT_TYPE_IDS( Obj self )
 {
   NextTypeID = -(1L << NR_SMALL_INT_BITS);
   CallbackForAllBags( FixTypeIDs );
+  CALL_0ARGS(FLUSH_ALL_METHOD_CACHES);
   return INTOBJ_INT(NextTypeID);
 }
 
 /****************************************************************************
 **
-*F  DoOperation( <name> ) . . . . . . . . . . . . . . .  make a new operation
+*F  DoOperation<N>Args( <oper>, ... ) . . . . . . . . . . .Operation Handlers
+**
+**  This section of the file provides handlers for operations. The main ones are
+**  DoOperation0Args..DoOperation6Args and the DoVerboseOperation tracing
+**  variants. Then there are variants for constructors. In the following section
+**  are handlers for attributes, properties and the operations related to them.
+**
+**  This code is being refactored (Oct 2017) to reduce repetition. It's efficiency
+**  now depends on the C compiler inlining some quite large functions and then doing
+**  constant folding to effectively produce a specialised version of the main function
```

Comment:
(Almost) all of these comments are limited to 78 columns; this one is not; perhaps rewrap?

Or, leave it as is, whatever, I am just glad there is a comment, thanks :-). I merely wanted to point this out, like any other nitpick :-).